ksqldb-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -127,9 +127,13 @@ protected Void visitQuery(final Query node, final Integer indent) {
       }
 
       if (!node.isPullQuery()) {
-        append(indent, "EMIT ");
-        append(indent, node.getResultMaterialization().toString())
-            .append('\n');
+        if (node.getResultMaterialization().isPresent()) {
+          append(indent, "EMIT ");
+          append(indent, node.getResultMaterialization()
+              .orElse(ResultMaterialization.CHANGES)
```

Comment:
nit: can just be `.get()` we know that it's present from the check above
```suggestion
              .get()
```